docs/building-a-form/common-definitions.md

```diff
@@ -45,22 +47,25 @@ Available definitions are:
 ### Autosuggest
 
 A common type-ahead widget that lets a user type in values and narrow down a longer list of options. It is most commonly used with an `enum` of the available options as shown here. Define the uiSchema by calling the function that you import. You can pass an object with additional uiSchema properties.
+
 ```js
 import { uiSchema as autosuggestUI } from 'us-forms-system/lib/js/definitions/autosuggest';
 
-schema: {
-  type: 'object',
-  properties: {
-    officeLocation: {
-      type: 'string',
-      enum: [
-        'LA', 'NY', 'CH'
-      ],
-      enumNames: [
-        'Los Angeles',
-        'New York',
-        'Chicago'
-      ]
+function schema() {
+  return {
+    type: 'object'
+    properties: {
+      officeLocation: {
+        type: 'string',
+        enum: [
+          'LA', 'NY', 'CH'
+        ],
+        enumNames: [
+          'Los Angeles',
+          'New York',
+          'Chicago'
+        ]
+      }
     }
   }
 },
```

Comment:
The usage for the `uiSchema` below also needs to change, where instead it would look like this:

```
uiSchema: {
  officeLocation: autosuggestConfig.uiSchema(
    ...
  )
}
```

(what is contained in the `...` is exactly as it is now (lines 74-81), it's just where the uiSchema function is called that needs to change)